.travis.yml

```diff
@@ -0,0 +1,61 @@
+sudo: false
+dist: trusty
```

Comment:
Not `xenial`?

Reply:
For whatever reason, WP-CLI still uses `trusty`, and I find no mention of switching to `xenial` at https://github.com/wp-cli/scaffold-command/issues. Interestingly, I don't find any repos in our organization that've switched to `xenial`.